consensus/service/src/bin/main.rs

```diff
@@ -68,6 +70,7 @@ fn main() -> Result<(), ConsensusServiceError> {
         &config.client_responder_id,
         &cached_key,
         &fee_map,
+        BlockVersion::try_from(config.block_version).unwrap(),
```

Comment:
Hmm, is it not possible to have `config.block_version` be of type `BlockVersion`?

Reply:
looks like it works, i just need `BlockVersion` to implement `FromStr`